crates/table/src/layout.rs

```diff
@@ -268,7 +267,7 @@ pub struct SumTypeVariantLayout {
     pub name: Option<Box<str>>,
 }
 
-/// Variants of [`BuiltinType`] which do not require a `VarLenRef` indirection,
+/// Scalar types which do not require a `VarLenRef` indirection,
 /// i.e. bools, integers and floats.
```

Comment:
```suggestion
/// Bools, integers and floats.
```

Reply:
Ok that's fine. But what scalar types **do** require a `VarLenRef`? If strings aren't considered scalars, then I don't think there are any.